### PR TITLE
ENG 1266 Make logs scroll to the bottom

### PR DESCRIPTION
**Pull request by MikeFuster**
https://user-images.githubusercontent.com/85583835/148229883-a8899f09-1e5b-46b3-9107-1132f0a5c691.mov

### CODE REVIEW

**Review by fperaltalitebox — APPROVED** [review]

**Comment by bennbollay** [comment]
Doesn't work for "Order=Ascending" panels:

https://user-images.githubusercontent.com/3607121/148268261-fdf24538-3989-42de-87ad-0f104c771191.mov

**Review by bennbollay — CHANGES_REQUESTED** [review]
Also doesn't work for Ascending:

https://user-images.githubusercontent.com/3607121/148268618-e75d83fd-a01d-4d25-bf61-1b30c70ece3c.mov

File: public/app/plugins/panel/logs/LogsPanel.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ export const LogsPanel: React.FunctionComponent<LogsPanelProps> = ({
 
   useLayoutEffect(() => {
     if (isAscending && logsContainerRef.current) {
-      setScrollTop(logsContainerRef.current.offsetHeight);
+      setScrollTop(logsContainerRef.current.scrollHeight);
     } else {
       setScrollTop(0);
     }
```
Comment (bennbollay) [inline public/app/plugins/panel/logs/LogsPanel.tsx:75]: Does this work on both ascending and descending log panels?

Comment (fperaltalitebox) [reply public/app/plugins/panel/logs/LogsPanel.tsx:75]: Hey @bennbollay! The last changes to the scroll feature is in [the iframe integration branch ](https://github.com/fusebit/grafana/pull/6 ). I asked to @MikeFuster to move only the scroll code to this branch so we can test and debug it atomically! It should be done in a brief 👍

**Pull request closed without merging**